iOS-TvShows/iOS-TvShows/Common/Extensions/UIViewController+Extensions.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  UIViewController+Extensions.swift
+//  iOS-TvShows
+//
+//  Created by Filip Ban on 09.11.2021..
+//
+
+import UIKit
+
+extension UIViewController {
+
+    var leftBarButton: UIBarButtonItem? {
+        navigationItem.leftBarButtonItem
+    }
+
+    var rightBarButton: UIBarButtonItem? {
+        navigationItem.rightBarButtonItem
+    }
+
+    func hideNavigationBar() {
```

Comment:
Please pass the animated flag here since this method should only be used from `viewWillAppear` otherwise you'll have weird glitches on UI. Also even tho we tend to have less code to have better readability, think about onboarding new colleague on project of yours where you are using this approach - it's a bit non-iOS to have hideNavigationBar() on view controller since all iOS developers know they should hide it on navigation controller. Having these kinds of method short your code for few character but increase the complexity of understanding your code for common iOS devs